server/sonar-server/src/main/java/org/sonar/server/issue/ws/BulkChangeAction.java

```diff
@@ -275,12 +277,18 @@ public DefaultIssue issue() {
     public IssueChangeContext issueChangeContext() {
       return changeContext;
     }
+
+    @Override
+    public String issueOrganizationUuid() {
+      return issueOrganizationUuid;
+    }
   }
 
   private class BulkChangeData {
     private final Map<String, Map<String, Object>> propertiesByActions;
     private final boolean sendNotification;
     private final Collection<DefaultIssue> issues;
+    private final Map<String, String> organizationUuidsByProjectUuid;
```

Comment:
minor: rename to `organizationUuidByProjectUuid` as it's a map and not a multimap